libs/common/src/main/kotlin/event/BehandlingGrunnlagEndret.kt

```diff
@@ -1,7 +1,6 @@
 package no.nav.etterlatte.libs.common.event
 
 interface IBehandlingGrunnlagEndret {
```

Comment:
Denne bør renames. Brukes fortsatt for andre hendelser, så kan ikke slettes, men hendelsen behandlinggrunnlagendret finnes jo ikke lenger. `IBehandlingHendelse`, kanskje?